### DISCUSSION TITLE
Corners radius possible ?

### DISCUSSION THREAD

**Discussion by xkain** [tsujan/Kvantum#936] [Q&A]
Hello,

 I am currently using the fork of Breeze Lightly but it is no longer maintained and not ported to Qt6 so I am seriously thinking about using Kvantum (which works in Qt6 and thank you for that), so I started to create my own theme but I don't know if it's possible with Kvantum to make rounded corners in Dolphin like in the photo example, if it's not currently possible, do you think it would be possible to implement this option ?

Thank you very much for your work
![Screenshot_20240206_195544](https://github.com/tsujan/Kvantum/assets/93977698/11113661-6cee-4677-8d96-7189a0aa9b3a)

**Comment by tsujan** [comment]
It's one of those places that only a small degree of rounding is possible. The following screenshot shows what I mean:

![roundness](https://github.com/tsujan/Kvantum/assets/981076/ef48b603-2887-4c99-9fd1-2bd8fcc70d82)

**Reply by xkain** [reply]
On Lightly the maximum possible radius is 8px (I don't know how much that is in degrees), it's not much, but it makes a huge difference in modernity compared to the original Breeze theme.

Could you tell me what theme you used in your example so that I can see how to reproduce this radius or guide myself on how to put it?

Thank you very much for answering me so quickly, it's very kind of you and sorry for my bad English

**Reply by tsujan** [reply]
> Could you tell me what theme you used in your example?

The screenshot was taken with KvCurves3d1.

It's about the values of `frame.top/bottom/left/right` under `[GenericFrame]` and also the SVG file. Those values shouldn't be chosen to be large (4 is enough); hence the limitation is this case ("frame expansion" isn't supported here -- see the doc).

Everything is explained in the doc (two files), but I don't pretend that making a Kvantum theme is an easy job; conversely, since there are so many parameters, it needs patience.

**Reply by xkain** [reply]
Thank you for your help and your responsiveness, i will study the documentation.
